### PR TITLE
Completed Dict_lab

### PR DESCRIPTION
**Pull request by FredBallyns**

### CODE REVIEW

**Comment by FredBallyns** [comment]
Finished Mailroom part2

File: students/FredB/session04/dict_lab.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+dict_data["Fruit"] = "Mango"
+
+print(dict_data)
+print(dict_data.keys())
+print(dict_data.values())
+
+print("cake" in dict_data)
+
+print("Fruit" in dict_data)
```
Comment (nairraghav) [inline students/FredB/session04/dict_lab.py:22]: Close, we're not necessarily looking to see if "Fruit" exists as a key but that "Mango" exists as a value. In this specific case, this is the same

File: students/FredB/session04/dict_lab.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+#Dictionaries 2
+dict_count={}
+for i in dict_data.keys():
```
Comment (nairraghav) [inline students/FredB/session04/dict_lab.py:26]: When you want to search through the keys of a dict, you don't need to specify the .keys()

Ex: for key in dict_data:

File: students/FredB/session04/dict_lab.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+
+#Sets2
+letter_set = set("Python")
+print(letter_set)
+letter_set.add("i")
+print(letter_set)
+frozenset = set("marathon")
+print(letter_set.union(frozenset))
+print(letter_set.intersection(frozenset))
```
Comment (nairraghav) [inline students/FredB/session04/dict_lab.py:49]: Nice job!

File: students/FredB/session04/mailroom2.py [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+        answer = input("""What would you like to do?
+Pick one:
+1: Send a Thank You
+2: Create a Report
+3: Send letters to all donors
+4: Quit
+>>>""")
+        #print(answer)
+        clear_screen()
+        if answer == "1" or answer.lower() == "send a thank you":
```
Comment (nairraghav) [inline students/FredB/session04/mailroom2.py:94]: Try to use the dict switch here!

**Comment by nairraghav** [comment]
Great job Fred, this looks great!

**Pull request merged**